.gitlab/common/shared.yml

```diff
@@ -34,3 +34,8 @@
   - *get_artifactory_bypass_win
   - *get_artifactory_token_win
   - if (($Env:USE_CACHING_PROXY_PYTHON -eq "true") -and ($Env:ARTIFACTORY_BYPASS -eq "false")) { $PIP_INDEX_URL="https://${Env:ARTIFACTORY_USERNAME}:${ARTIFACTORY_TOKEN}@${Env:ARTIFACTORY_URL}/${Env:ARTIFACTORY_PYPI_PATH}" }
+
+.setup_deb_signing_key: &setup_deb_signing_key
+  - DEB_GPG_KEY=$($CI_PROJECT_DIR/tools/ci/aws_ssm_get_wrapper.sh $DEB_GPG_KEY_SSM_NAME)
+  - printf -- "${DEB_GPG_KEY}" | gpg --import --batch
+  - export DEB_SIGNING_PASSPHRASE=$($CI_PROJECT_DIR/tools/ci/aws_ssm_get_wrapper.sh $DEB_SIGNING_PASSPHRASE_SSM_NAME)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏To avoid any leaks maybe adding a `set +x` would be safer ? Using these kind of jobs as reference could be dangerous leaks-wise, wdyt?

```suggestion
.setup_deb_signing_key: &setup_deb_signing_key
  - set +x
  - DEB_GPG_KEY=$($CI_PROJECT_DIR/tools/ci/aws_ssm_get_wrapper.sh $DEB_GPG_KEY_SSM_NAME)
  - printf -- "${DEB_GPG_KEY}" | gpg --import --batch
  - export DEB_SIGNING_PASSPHRASE=$($CI_PROJECT_DIR/tools/ci/aws_ssm_get_wrapper.sh $DEB_SIGNING_PASSPHRASE_SSM_NAME)
```

Reply:
Very valid point, thanks a lot! And I just realized thee was a suggestion I could have accepted instead of doing it manually :sweat_smile: Oh well...